media/js/xeditable/bootstrap-editable-edtf.js

```diff
@@ -97,12 +97,14 @@
                      .removeClass('has-error')
                      .find('.help-block').hide();
 
-            if (this.$millenium1.val().length < 1) {
+            if (this.$millenium1.val().length < 1 ||
+                    this.$millenium2.val() < 1) {
```

Comment:
It looks like you meant this check to be on `this.$millenium1`, not `this.$millenium2`.